pkg/model/manifest.go

```diff
@@ -626,7 +626,7 @@ func getOktetoManifest(devPath string) (*Manifest, error) {
 
```

Comment:
If there is an i/o error reading the file, obtaining the bytes *and* the file exists, then we must not fallback.

I'm not sure we can tell directly if the file exists or not based on the error so we may need to first do a fs or os Stat call or something that is specific to the file not existing. If the file exists *and* an error occurrs then we can't fallback to v1 behavior.

Reply:
Added implementation and test that notifies of an error due to an invalid manifest if the file exists but there is no content. Also added a check if the manifest exists (if it doesn't exist, fallback). After talking with @jLopezbarb  about this we have concluded that for the moment we cover the cases where the user should be notified by an invalid manifest or otherwise fallback to buildv1. Refactoring these methods is tricky as they are used in almost every command.